qcfractal/util.py

```diff
@@ -0,0 +1,24 @@
+"""
+Utility functions for QCFractal.
+"""
+
+import socket
+
+
+def find_port() -> int:
+    sock = socket.socket()
+    sock.bind(('', 0))
+    host, port = sock.getsockname()
+    return port
+
+def port_open(ip: str, port: int) -> bool:
```

Comment:
Would you be willing to change this to `is_port_open`? No need to actually do this, just a flavor thought.

Reply:
Sure.